plugins/extractcode-7z-macosx_10_9_intel/setup.cfg

```diff
@@ -1,5 +1,12 @@
 [metadata]
 license_file = LICENSE.txt
-
+[bdist_wheel]
```

Comment:
This is not needed for now if you run a `pip install -e ....` instead

Reply:
pip install -e ... is not working on python3.I have tried.It saying that "Collecting extractcode-libarchive (from scancode-toolkit===3.0.2)
Could not find a version that satisfies the requirement extractcode-libarchive (from scancode-toolkit===3.0.2) (from versions: )"